IFC4x3/Properties/r/Reference_09uhUrGP96Dg7LCTKoeQZI/DocProperty.xml

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Reference_09uhUrGP96Dg7LCTKoeQZI" Name="Reference" UniqueId="09e2b7b5-4192-4636-a1d5-31d532a1a8d2" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Reference_09uhUrGP96Dg7LCTKoeQZI" Name="Reference" UniqueId="09e2b7b5-4192-4636-a1d5-31d532a1a8d2" PrimaryDataType="IfcIdentifier" />
```

Comment:
Did you check of all of these were `IfcIdentifier` before? This one was `IfcLabel`.

Reply:
IfcIdentifier in leading edge docs from BSI online, it is deprecated anyway so no issue